cms/search/tests/test_api_views.py

```diff
@@ -43,11 +44,29 @@ def setUpTestData(cls):
             IndexPageFactory(slug="custom-slug-1"),
         ]
 
+        cls.en = Locale.get_default()
+        if cls.en.language_code != "en-gb":
+            cls.en.language_code = "en-gb"
+            cls.en.save()
```

Comment:
given `en-gb` is the default language ([ref](https://github.com/ONSdigital/dis-wagtail/blob/main/cms/settings/base.py#L365)), what scenario does this cover?